translations/en-us.yaml

```diff
@@ -45,6 +45,7 @@ newMeasureConcept:
   roadSignConcept: Road Sign
   noData: No results
   create: Create new measure
+  required: Required
```

Comment:
I think these don't need to be duplicated. I can't imagine a reason why you want multiple variantions for this. I know you added them to be consistent though :smile:.

Reply:
well i proposed to have something like 
common:
   required: xxx
   
  But the team kinda ignore me so I'm following the trends :p

Reply:
Yea, I think Anita created a utilities top level thing in her PR. Maybe you can check that out.

```yml
utilities:
  required: Required
```